### PR TITLE
Update templating.rst - Asset absolute url fix

### PR DESCRIPTION
**Pull request by gbalcewicz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 3.0+ |
| Fixed tickets | N/A |

Fixed the description of using absolute_url for the asset.

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

File: book/templating.rst [hunk 1]
```diff
@@ -1076,14 +1076,14 @@ configuration option.
...
-If you need to set a version for a specific asset, you can set the ``version`` argument
+If you need to set a version for a specific asset, you can use the ``asset_version`` function
 if you are using Twig (or the fourth argument if you are using PHP) to the desired version:
 
 .. configuration-block::
 
     .. code-block:: html+jinja
 
-        <img src="{{ asset('images/logo.png', version='3.0') }}" alt="Symfony!" />
+        <img src="{{ asset_version('images/logo.png', '3.0') }}" alt="Symfony!" />
```
Comment (xabbuh) [inline book/templating.rst]: Actually, you will now have to define the version in the asset configuration for the affected package and then call `asset_version()` with the package name (see http://symfony.com/blog/new-in-symfony-2-7-the-new-asset-component).

**Comment by gbalcewicz** [comment]
I've reverted changes made for asset version and left only for the absolute url.

**Comment by xabbuh** [comment]
Thank you @gbalcewicz.

**Pull request closed without merging**